### DISCUSSION TITLE
Add class name obfuscation

### DISCUSSION THREAD

**Discussion by gyerts** [tailwindlabs/tailwindcss#10296] [Ideas]
It is good to have class name obfuscation while production build. It protects our code from copy past and even more if obfuscated name will have shorter name, that means that style boundle will have less size. eg:
dev mode
```
.flex-col {
  flex-direction: column;
}
```
prod mode
```
.a1d {
  flex-direction: column;
}
```

**Comment by gyerts** [comment]
Now I see that such discussion already exists. But still there is no native solution exists.

**Comment by sonofmagic** [comment]
Hi, i write a plugin to obfuscate tailwindcss classes. you can have a try.
npm: [unplugin-tailwindcss-mangle](https://www.npmjs.com/package/unplugin-tailwindcss-mangle)
source code: https://github.com/sonofmagic/tailwindcss-mangle/tree/main/packages/unplugin-tailwindcss-mangle

Now, this plugin support `webpack` and `vite`.it work's when `webpack/vite` build your app.
